### DISCUSSION TITLE
Having trouble setting up

### DISCUSSION THREAD

**Discussion by patateld** [DIGITALCRIMINALS/OnlyFans#428] [Q&A]
I'm pretty illiterate when it comes to code so thanks in advance for any help.
"pip install poetry" gives a few warnings such as 
 WARNING: The script poetry.exe is installed in 'C:\Users\\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\Scripts' which is not on PATH.
which i have to guess are an issue since the next part "poetry install" gives the term 'poetry' is not recognized as the name of a cmdlet, function, script file, or operable program.
Despite this i've tried running "python start_ofd.py " which returns
Traceback (most recent call last):
  File "C:\Users\\Downloads\OnlyFans-7.6.1\start_ofd.py", line 7, in <module>
    from rich import panel
ModuleNotFoundError: No module named 'rich'
Again, thanks for any help

**Comment by neil-ui** [comment]
I'm having similar issues and am one step ahead of you as I solved that installing poetry bit by following the instructions in 
https://stackoverflow.com/a/30728643
However I now get the error trying to run 'poetry install' - 
'poetry' is not recognized as an internal or external command, operable program or batch file.
Which I can't find the answer to.
I had it working last week, but after getting Access Denied messages I reinstalled everything and now I can't get it running!
